quantmsrescore/idxmlreader.py

```
@@ -0,0 +1,466 @@
+
+
+class SpectrumStats:
+    """Statistics about spectrum analysis."""
+
+    missing_spectra: int = 0
+    empty_spectra: int = 0
+    ms_level_counts: DefaultDict[int, int] = defaultdict(int)
+    ms_level_dissociation_method = {}
+
+
+class IdXMLReader:
```

Ruff (0.8.2)

Redefinition of unused `IdXMLReader` from line 10

(F811)